src/main/java/beam/physsim/jdeqsim/AgentSimToPhysSimPlanConverter.java

```diff
@@ -142,6 +195,10 @@ private void setupActorsAndRunPhysSim(int iterationNumber) {
         linkStatsGraph.notifyIterationStarts(jdeqsimEvents, agentSimScenario.getConfig().travelTimeCalculator());
 
         log.info("JDEQSim Start");
+
+        checkForBrokenLinksInsideNetwork(jdeqSimScenario);
+        checkForErroneousLinkNodes();
```

Comment:
checkForErroneousLinkNodes - this check can be done just once - no need to do in every iteration.

Reply:
done